website/static/downloads/swagger.yaml

```diff
@@ -780,7 +621,126 @@ components:
       content:
         application/json:
           schema:
-            $ref: '#/components/schemas/error'
+            $ref: '#/components/schemas/errResult'
+    contractResp:
+      description: Single contract
+      content:
+        application/json:
+          schema:
+            $ref: '#/components/schemas/contractWrap'
+    contractPollResp:
+      description: |
+        Establishes persitent connection and sends json value through it to the requester as long as new updates appear.
+      headers: 
+        -X-Stream-Output:
+          schema:
+            type: integer
+            enum:
+             - 1
+      content:
+        application/json:
+          schema:
+            $ref: '#/components/schemas/contractWrap'
+    invitePollResp:
+      description: |
+        Establishes persitent connection and sends json value through it to the requester as long as new updates appear.
+      headers: 
+        -X-Stream-Output:
+          schema:
+            type: integer
+            enum:
+             - 1
+      content:
+        application/json:
+          schema:
+            $ref: '#/components/schemas/inviteWrap'
+    contractVerifyResp:
+      description: Start a drive verification
+      headers: 
```

Comment:
Could you delete lines 659-664, please

Reply:
ok done. I am merging the PR into master